File: problems/s/skocimis/skocimis.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    a, b, c = [int(val) for val in input().split()]
    space_1 = b - a
    space_2 = c - b
    if space_1 > space_2:
        print(space_1 - 1)
    else:
        print(space_2 - 1)

if __name__ == '__main__':
    main()

File: problems/s/skocimis/skocimis.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>

#define COMMAND_SIZE 101

int main(void) {
    int a, b, c, space_1, space_2;

    if (scanf("%d %d %d\n", &a, &b, &c) != 3) {
        printf("Failed to read a, b and c!");
    }
    
    space_1 = b - a;
    space_2 = c - b;

    if (space_1 > space_2) {
        printf("%d", space_1 - 1);
    }
    else {
        printf("%d", space_2 - 1);
    }

    return 0;
}